containers/integration_tests/docker-compose.yml

```diff
@@ -18,39 +40,61 @@ services:
       - ../..:/workspace
 
   aqueductcore:
-    image: aqueducthub/aqueductcore-dev:latest
-    container_name: aqueductcore-server
-    depends_on:
-      postgres:
-        condition: service_healthy
+    <<: *aqueduct-common
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.aqueduct.rule=Host(`aqueduct.localhost`)"
+    expose:
+      - 8000
 
-    environment:
-      EXPERIMENTS_DIR_PATH: /tmp/aqueduct_experiments
-      EXTENSIONS_DIR_PATH: /tmp/aqueduct_extensions
-      POSTGRES_USERNAME: admin
-      POSTGRES_PASSWORD: admin
-      POSTGRES_HOST: postgres
-      POSTGRES_PORT: 5432
-      POSTGRES_DB: aqueduct
+  aqueduct-worker:
+    <<: *aqueduct-common
+    labels:
+      - "traefik.enable=false"
+    command: aqueduct worker
 
-    volumes:
-      - /tmp/aqueduct_experiments:/tmp/aqueduct_experiments
-      - /tmp/aqueduct_extensions:/tmp/aqueduct_extensions
+  aqueduct-flower:
+    <<: *aqueduct-common
+    command: aqueduct flower --broker-api http://guest:guest@rabbitmq:15672/api/vhost
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.aqueduct-flower.rule=Host(`flower.localhost`)"
+    ports:
+      - 5555
```

Comment:
Considering the case here that this docker-compose file is going to be used for integration tests and the docker containers will be destroyed after the integration_tests scripts completes, do we need to run flower here ?

I think this can be removed

Reply:
Yeah, you're so correct on it, removed.